kong/api/routes/cluster_status.lua

```diff
@@ -1,18 +1,19 @@
-local clustering = require("kong.clustering")
-local kong = kong
+local endpoints = require "kong.api.endpoints"
 
 
 return {
-  ["/clustering/status"] = {
```

Comment:
Note: I'm fine with breaking this but if we do do this, please ensure that it is called out in the changelog.